plainly-plugin/src/node/settings/settings.ts

```diff
@@ -12,6 +12,7 @@ async function retrieveSettings(): Promise<Settings> {
     return JSON.parse(file);
   } catch (error) {
     console.error(`Failed to read settings: ${error}`);
+    fsPromises.writeFile(settingsPath, JSON.stringify(defaultSettings));
```

Comment:
why? before reading the file check if file exists and if not return default settings!?

```
async function retrieveSettings(): Promise<Settings> {
   if (fileNotExists) {
     return defaultSettings
   } 
   try {
   // evertying as is
```